website/client/src/components/groups/group.vue

```diff
@@ -25,11 +25,13 @@
         <div class="col-12 col-md-6">
           <div class="row icon-row">
             <div
-              class="col-4 offset-4"
               :class="{ 'offset-8': isParty }"
```

Comment:
I think the offset here might not be necessary anymore

Reply:
Yes, looks like that entire container div can go without any problems.